modules/install-guides/pages/install-guide-linux.adoc

```diff
@@ -0,0 +1,286 @@
+
+- A minimal distribution-spec conformant zot binary image is compiled with
+only a minimal set of code and libraries, reducing the attack surface. This
+option might be optimal for a registry embedded in a shipping product.
+
+=== Binary image file naming
+
+An executable binary image for zot is named using the target platform and
+architecture from the
+ <<supported-platforms-zot, Supported platforms and architectures>>
+ table. The general format of a binary image file name is one of these two:
+
+``zot-_os_-_architecture_``
+
+- A full zot binary image with all extensions has a filename of the
+form ``zot-_os_-_architecture_.`` For example, the full binary image for an
+Intel-based linux server is `zot-linux-amd64`.
+
+``zot-_os_-_architecture_-minimal``
+
+- A minimal distribution-spec conformant zot binary image has a filename of
+the form ``zot-_os_-_architecture_-minimal``. For example, the minimal binary
+image for an Intel-based linux server is `zot-linux-amd64-minimal`.
+
+== Installation
+
+=== Step 1: Get zot
+
+Using `wget,` download the appropriate zot binary image for your platform from the
+https://github.com/project-zot/zot/releases[zot GitHub project]. Download the image
+to the`/usr/bin/` directory and rename it to `zot,` as in this example:
+
+----
+$ wget -O /usr/bin/zot  https://github.com/project-zot/zot/releases/download/v1.4.2-rc2/zot-linux-amd64
+----
+
+.MIKE'S QUESTIONS
+****
+. Do we need to rename the binary or to chmod it to be executable?
+. What does -N do?
+****
+
+=== Step 2: Create a zot configuration file
+
+Create a zot configuration file as `/etc/zot/config.json`.
+
+See <<config_file,Configuration file options>> for an example file with
+options and recommendations. You can find other onfiguration file
+https://github.com/project-zot/zot/tree/main/examples[examples] in
+the zot GitHub project and in the _Zot Administrator Guide_.
+
+
+=== Step 3: Configure a local authentication account
+
+If you want to use local authentication with zot, create a `/etc/zot/htpasswd`
+file with an initial account entry using the `htpasswd` command as in this
+example:
+
+----
+$ htpasswd -bnB myUserName myPassword > /etc/zot/htpasswd
+----
+
+To add additional local users, use the `>>` redirect as in this example:
+
+----
+$ htpasswd -bnB myUserName2 myPassword2 >> /etc/zot/htpasswd
+----
+
+=== Step 4: Define the zot service
+
+Create a `/etc/systemd/system/zot.service` file to define the zot service
+in systemd.  The following is an example service file for zot:
+
+----
+[Unit]
+Description=OCI Distribution Registry
+Documentation=https://zotregistry.io/
+After=network.target auditd.service local-fs.target
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/zot serve /etc/zot/config.json
+Restart=on-failure
+User=zot
+Group=zot
+LimitNOFILE=500000
+MemoryHigh=30G
+MemoryMax=32G
+
+[Install]
+WantedBy=multi-user.target
+----
+
+NOTE: Be sure to make 'zot' the User and Group.
```

Comment:
Be sure to make `zot` service run with its own non-root user and group, for example as 'zot' following the service definition above.